airflow/api_connexion/schemas/dag_schema.py

```diff
@@ -17,10 +17,12 @@
 
 from typing import List, NamedTuple
 
+from itsdangerous import URLSafeSerializer
```

Comment:
Do we need to add this as a dependency in setup.py?

Reply:
We should, yes. It'll already be pulled in from flask/jinja, but we should record we use it directly.

Reply:
Should I add it as flask dependency e.g flask=['itsdangerous'] in setup.py?

Reply:
Add it directly in to this list https://github.com/apache/airflow/blob/master/setup.cfg#L81-L134